src/cpu_shader/flatten.rs

```diff
@@ -202,17 +149,68 @@ fn output_two_lines_with_transform(
     *line_ix += 2;
 }
 
-const MAX_QUADS: u32 = 16;
+fn flatten_arc(
+    path_ix: u32,
+    begin: Vec2,
+    end: Vec2,
+    center: Vec2,
+    angle: f32,
+    transform: &Transform,
+    line_ix: &mut usize,
+    lines: &mut [LineSoup],
+    bbox: &mut IntBbox,
+) {
+    let mut p0 = transform.apply(begin);
+    let mut r = begin - center;
+    let tol: f32 = 0.5;
```

Comment:
Is `0.5` the tolerance value we want to go with?

Reply:
That's a very good question and we should take a closer look. One other thing I've noticed in testing is the second longpathdash now exceeds memory limits, and we should really understand why.